index.html

```diff
@@ -29,6 +29,10 @@
           margin: 0.5rem;
           width: calc(100%/4);
         }
+        #search{
```

Comment:
i think it needs to be .search instead of #search since you're using className instead of id

Reply:
sorry I didn't see this comment, I did not fix in last pull request